managed/ui/public/swagger-ui/index.html

```diff
@@ -4,74 +4,23 @@
   <head>
     <meta charset="UTF-8">
     <title>Swagger UI</title>
-    <link rel="stylesheet" type="text/css" href="./swagger-ui.css" />
-<!--    <link rel="icon" type="image/png" href="./favicon-32x32.png" sizes="32x32" />-->
-<!--    <link rel="icon" type="image/png" href="./favicon-16x16.png" sizes="16x16" />-->
+    <link rel="icon" type="image/png" href="./favicon-32x32.png" sizes="32x32" />
+    <link rel="icon" type="image/png" href="./favicon-16x16.png" sizes="16x16" />
     <link rel="icon" type="image/ico" href="../favicon.ico" sizes="256x256" />
     <link rel="icon" type="image/ico" href="../favicon.ico" />
-    <style>
-      html
-      {
-        box-sizing: border-box;
-        overflow: -moz-scrollbars-vertical;
-        overflow-y: scroll;
-      }
-
-      *,
-      *:before,
-      *:after
-      {
-        box-sizing: inherit;
-      }
-
-      body
-      {
-        margin:0;
-        background: #fafafa;
-      }
+    <link rel="stylesheet" href="./styles.min.css">
+    <script src="./web-components.min.js"></script>
     </style>
   </head>
 
   <body>
     <div id="swagger-ui"></div>
+    <elements-api
+      apiDescriptionUrl="/swagger.json"
+      router="hash"
+      layout="sidebar"
+    />
 
-    <script src="./swagger-ui-bundle.js" charset="UTF-8"> </script>
-    <script src="./swagger-ui-standalone-preset.js" charset="UTF-8"> </script>
-    <script>
-    function getCSRF() {
-      var name = "csrfCookie=";
-      var ca = document.cookie.split(';');
-      for(var i=0; i<ca.length; i++) {
-          var c = ca[i];
-          while (c.charAt(0) === ' ') c = c.substring(1);
-          if (c.indexOf(name) !== -1) return c.substring(name.length,c.length);
-      }
-      return "";
-    }
-
-    window.onload = function() {
-      // Begin Swagger UI call region
-      const ui = SwaggerUIBundle({
-        url: "/swagger.json",
-        dom_id: '#swagger-ui',
-        deepLinking: true,
-        presets: [
-          SwaggerUIBundle.presets.apis,
-          SwaggerUIStandalonePreset
-        ],
-        plugins: [
-          SwaggerUIBundle.plugins.DownloadUrl
-        ],
-        layout: "StandaloneLayout",
-        requestInterceptor: function(req) {
```

Comment:
See this. I had fixed the CSRF issue so swagger-ui could be used to try out post and put requests as well. 
Does stoplight take care of it internally?

Reply:
Yes. I have tested delete and post API.

Reply:
Worked Added screenshot.